src/ActivityLogger.php

```diff
@@ -72,6 +72,14 @@ public function causedBy($modelOrId)
         return $this;
     }
 
+    public function causedByAnonymous()
```

Comment:
could you add an alias `byAnonymous()` like the `by()`, `on()` methods?

Reply:
I added them in the follow up PR. Do you also want me to add separate tests for these aliases?

Reply:
https://github.com/spatie/laravel-activitylog/pull/605/commits/d3efff378ff86cd35bd6690f93e37f9a87b7be5e

Reply:
You already have two tests, so just use the main method in the one and the alias in the other test. :)

Reply:
I added `causedByAnonymous()` in the first test, and `byAnonymous()` in the second test (testing overriding the currently logged in user).

Reply:
https://github.com/spatie/laravel-activitylog/pull/605/commits/3f5be6cc8a79beee388477ee5df436435222ef92